layouts/_default/baseof.html

```diff
@@ -31,9 +31,10 @@
     {{ if eq .Type "generator" -}}
     <!-- TODO: Implement this correctly! <link rel="preload" as="script" href="{{ "js/pdfworker.gen.js" | absURL }}"> -->
     {{- end }}
-    <link rel="stylesheet" href="https://static.dacdn.de/fonts/fira-sans.css" integrity="sha384-tdesaAgh/bq3iJZEqjfYp6o6sVopRLnw3wjZKW9gLPpNd26Pdgbur5PChCva/Z+2" crossorigin="anonymous">
-    <link rel="stylesheet" href="https://static.dacdn.de/fonts/league-spartan.css" integrity="sha384-n561KFrMi6x/rwAxQhsRLmxmF+oKREXWnteNZCFbPbU1nODnUri2U+r9p/lCJ918" crossorigin="anonymous">
-    <link rel="stylesheet" href="{{ "css/style.gen.css" | absURL }}" />
+    <link rel="stylesheet" href="https://static.dacdn.de/fonts/fira-sans.css" integrity="sha384-U8TLqRd5TXsbHc89iPjGqbk/4KrbgxDjzI37GGTuQ30EdFX0pgdZZzPJPLkL2iJR" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://static.dacdn.de/fonts/league-spartan.css" integrity="sha384-8xT2dJ0AqzYTGWPC27D6SB6ndu+drRpQGL4BAxDYQvC87Adakg1LrWQY5t84tkkO" crossorigin="anonymous">
```

Comment:
The rebase failed here. THE SRIs have changed:

https://github.com/datenanfragen/website/blob/9195f52c9366844e6fe8c08fe88952776eb29c05/layouts/_default/baseof.html#L34-L35